ws-style.gemspec

```diff
@@ -23,6 +23,7 @@ Gem::Specification.new do |s|
   s.required_ruby_version = '>= 2.5.7'
 
   s.add_dependency 'rubocop', '>= 1.0.0'
+  s.add_dependency 'rubocop-inflector'
```

Comment:
This doesn't appear to be an official rubocop gem and hasn't been updated in 2 years. Just wondering if it makes sense to add a dependency on it?

Reply:
There is a ticket open to incorporate, however it hasn't been updated in over a year.
The actual configuration is just a single method override, so we can just incorporate it directly into ws-style.

Reply:
@andrewthauer implemented this directly into ws-style so we don't need to add a dependency 🎉